app/assets/scripts/components/spotlight/single/index.js

```diff
@@ -289,11 +307,29 @@ function mapStateToProps (state, props) {
       } else {
         return {
           ...l,
+          domain: layerProps[l.type] ? layerProps[l.type].domain : l.domain,
           enabled: l.id === 'nightlights-hd',
```

Comment:
We should find a better way to define what are the default enabled layers per spotlight. Maybe it is time to move the spotlights overrides to a separate file

Reply:
i agree, as well with the properties i've added here (specifying what types of detection data are available per spotlight). might be good to have one config file that enumerates this.